src/PlatformBar.js

```diff
@@ -2,31 +2,24 @@ import React, { Component } from 'react';
 import CheckBox from './CheckBox'
 import './styles/PlatformBar.css';
 
-// Requires 'state' to keep track of consoles the user selects?
 export default class ConsoleBar extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {showPlatforms: false};
```

Comment:
Woot woot! Platform is now toggling checkboxes.
We're getting close to **MVP**!